data/sql/updates/pending_db_world/rev_1588338688279133200.sql

```diff
@@ -0,0 +1,45 @@
+UPDATE `creature_template` SET `mindmg` = 757, `maxdmg` = 1098, `DamageModifier` = 1.03 WHERE `entry` = 18619;
+UPDATE `creature_template` SET `mindmg` = 733, `maxdmg` = 1015, `DamageModifier` = 1.03 WHERE `entry` = 17626;
+UPDATE `creature_template` SET `mindmg` = 970, `maxdmg` = 1371, `DamageModifier` = 1.03 WHERE `entry` = 18609;
+UPDATE `creature_template` SET `mindmg` = 568, `maxdmg` = 814, `DamageModifier` = 1.03 WHERE `entry` = 17395;
+UPDATE `creature_template` SET `mindmg` = 718, `maxdmg` = 1087, `DamageModifier` = 1.03 WHERE `entry` = 18617;
+UPDATE `creature_template` SET `mindmg` = 780, `maxdmg` = 1084, `DamageModifier` = 1.03 WHERE `entry` = 17624;
+UPDATE `creature_template` SET `mindmg` = 1020, `maxdmg` = 1444, `DamageModifier` = 1.03 WHERE `entry` = 18611;
+UPDATE `creature_template` SET `mindmg` = 568, `maxdmg` = 867, `DamageModifier` = 1.03 WHERE `entry` = 17414;
+UPDATE `creature_template` SET `mindmg` = 757, `maxdmg` = 1098, `DamageModifier` = 1.03 WHERE `entry` = 18618;
+UPDATE `creature_template` SET `mindmg` = 587, `maxdmg` = 867, `DamageModifier` = 1.03 WHERE `entry` = 17398;
+UPDATE `creature_template` SET `mindmg` = 1019, `maxdmg` = 1443, `DamageModifier` = 1.03 WHERE `entry` = 18612;
+UPDATE `creature_template` SET `mindmg` = 733, `maxdmg` = 1015, `DamageModifier` = 1.03 WHERE `entry` = 18894;
+UPDATE `creature_template` SET `mindmg` = 930, `maxdmg` = 1371, `DamageModifier` = 1.03 WHERE `entry` = 21645;
+UPDATE `creature_template` SET `mindmg` = 350, `maxdmg` = 487, `DamageModifier` = 1.03 WHERE `entry` = 17653;
+UPDATE `creature_template` SET `mindmg` = 520, `maxdmg` = 722, `DamageModifier` = 1.03 WHERE `entry` = 18620;
+UPDATE `creature_template` SET `mindmg` = 697, `maxdmg` = 965, `DamageModifier` = 1.03 WHERE `entry` = 17399;
+UPDATE `creature_template` SET `mindmg` = 936, `maxdmg` = 1325, `DamageModifier` = 1.03 WHERE `entry` = 18614;
+UPDATE `creature_template` SET `mindmg` = 142, `maxdmg` = 210, `DamageModifier` = 1.03 WHERE `entry` = 17401;
+UPDATE `creature_template` SET `mindmg` = 153, `maxdmg` = 216, `DamageModifier` = 1.03 WHERE `entry` = 18605;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 975, `maxdmg` = 1355, `DamageModifier` = 1.01 WHERE `entry` = 17381;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1060, `maxdmg` = 1503, `DamageModifier` = 1.01 WHERE `entry` = 18621;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 636, `maxdmg` = 914, `DamageModifier` = 1.01 WHERE `entry` = 17380;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 1246, `maxdmg` = 1810, `DamageModifier` = 1.01 WHERE `entry` = 18601;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 636, `maxdmg` = 914, `DamageModifier` = 1.01 WHERE `entry` = 17377;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 830, `maxdmg` = 1206, `DamageModifier` = 1.01 WHERE `entry` = 18607;
```

Comment:
update the type_flags to what I said in previous PR pls

Reply:
Sorted.